model/project_parser.go

```diff
@@ -547,11 +553,30 @@ func GetProjectFromBSON(data []byte) (*Project, error) {
 // LoadProjectInto loads the raw data from the config file into project
 // and sets the project's identifier field to identifier. Tags are evaluated. Returns the intermediate step.
 // If reading from a version config, LoadProjectForVersion should be used to persist the resulting parser project.
-func LoadProjectInto(data []byte, identifier string, project *Project) (*ParserProject, error) {
+// opts is used to look up files on github if the main parser project has an Include.
+func LoadProjectInto(ctx context.Context, data []byte, opts GetProjectOpts, identifier string, project *Project) (*ParserProject, error) {
 	intermediateProject, err := createIntermediateProject(data)
 	if err != nil {
 		return nil, errors.Wrapf(err, LoadProjectError)
 	}
+	if intermediateProject.Include != nil {
+		for _, path := range intermediateProject.Include {
+			opts.RemotePath = path.FileName
+			yaml, err := retrieveFile(ctx, opts)
```

Comment:
One other thought! If path has a module, we'll need to lookup the module from the intermediateProject and use that branch/repo/revision instead, in which case maybe retrieveFile should accept something different than ProjectRef (you can add a ticket to add handling for modules though if you'd like!)